src/SourceBuild/content/eng/Versions.props

```diff
@@ -25,13 +25,15 @@
     -->
     <PrivateSourceBuiltSdkVersion>9.0.100-preview.4.24179.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-preview.4.24179.1</PrivateSourceBuiltArtifactsVersion>
-    <!-- msbuild -->
-    <MicrosoftBuildVersion>17.8.3</MicrosoftBuildVersion>
-    <!-- runtime -->
-    <MicrosoftExtensionsFileSystemGlobbingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsFileSystemGlobbingVersion>
-    <MicrosoftExtensionsLoggingConsoleVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingConsoleVersion>
-    <MicrosoftExtensionsLoggingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingVersion>
-    <!-- command-line-api -->
+    <!-- command-line-api dependencies -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
+    <!-- msbuild dependencies -->
+    <MicrosoftBuildVersion>17.8.3</MicrosoftBuildVersion>
+    <!-- runtime dependencies -->
+    <MicrosoftExtensionsFileSystemGlobbingVersion>8.0.0</MicrosoftExtensionsFileSystemGlobbingVersion>
```

Comment:
@ellahathaway - What impact if any will this have on the BinaryTooling Scenarios?

Reply:
I ran this locally with both 8.0 and 9.0 versions, and the BinaryTooling worked in both scenarios.

Reply:
Yes, I tested this. The binary tool imports Versions.props / Directory.Packages.props so it has access to the versions.